cabal-testsuite/PackageTests/Path/Single/cabal.test.hs

```diff
@@ -1,3 +0,0 @@
-import Test.Cabal.Prelude
-
-main = cabalTest . void $ cabal "path" ["--installdir"]
```

Comment:
Is it still possible to extract a single value as plain string, like `cabal path --store-dir`, so I can do stuff like:
```
cd $(cabal path --store-dir)
```

Reply:
Yes, that should work exactly like this.